ledger/ledger.go

```diff
@@ -278,43 +305,43 @@ func (l *Ledger) verifyMatchingGenesisHash() (err error) {
 	return
 }
 
-func openLedgerDB(dbPathPrefix string, dbMem bool, cfg config.Local, log logging.Logger) (trackerDBs trackerdb.Store, blockDBs db.Pair, err error) {
-	// Backwards compatibility: we used to store both blocks and tracker
-	// state in a single SQLite db file.
-	if !dbMem {
-		commonDBFilename := dbPathPrefix + ".sqlite"
-		_, err = os.Stat(commonDBFilename)
-		if !os.IsNotExist(err) {
-			// before launch, we used to have both blocks and tracker
-			// state in a single SQLite db file. We don't have that anymore,
-			// and we want to fail when that's the case.
-			err = fmt.Errorf("a single ledger database file '%s' was detected. This is no longer supported by current binary", commonDBFilename)
-			return
-		}
```

Comment:
Want to just note it here for others -- I am removing this very old (pre-launch) check. it is >4y old, so any node with these files wouldn't really be expected to be operational anyway.